src/components/DisplayCards.tsx

```diff
@@ -107,5 +107,20 @@ export const DisplayCards = ({
 			</VStack>
 		);
 	if (index === 4) return <ProfileCard profile={user.contact.contacts[0]} />;
+	if (index === 6)
```

Comment:
why 6 and not 5?

It probably has to do with the final return of the config page :eyes:

Reply:
Fixed, used 5